ElectronClient/plugins/GotoAnything.jsx

```diff
@@ -368,22 +361,19 @@ class Dialog extends React.PureComponent {
 	input_onKeyDown(event) {
 		const keyCode = event.keyCode;
 
-		if (this.state.results.length > 0 && (keyCode === 40 || keyCode === 38)) { // DOWN / UP
+		if ((keyCode === 40 || keyCode === 38) && this.state.results.length > 0) { // DOWN / UP
 			event.preventDefault();
 
-			const inc = keyCode === 38 ? -1 : +1;
 			let index = this.selectedItemIndex();
 			if (index < 0) return; // Not possible, but who knows
 
-			index += inc;
-			if (index < 0) index = 0;
-			if (index >= this.state.results.length) index = this.state.results.length - 1;
-
-			const newId = this.state.results[index].id;
+			index = Math.min(
+				index + (keyCode === 38 ? -1 : +1),
+				this.state.results.length - 1,
+			);
```

Comment:
As I understand, the new code does the same as the previous one, so please undo these changes. It makes it easier to review pull request if we limit to only the necessary changes.